packages/@webex/plugin-encryption/src/encryption/index.ts

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable require-jsdoc */
+/* eslint-disable no-unused-expressions */
+import {WebexPlugin} from '@webex/webex-core';
+import {proxyEvents} from '@webex/common';
+import {EventEmitter} from 'events';
+
+import {IEncryption} from './encryption.types';
+import {ENCRYPTION} from './constants';
+
+/**
+ * @description Encryption APIs for KMS
+ * @class
+ */
+class Cypher extends WebexPlugin implements IEncryption {
+  namespace = ENCRYPTION;
+
+  constructor(...args: any[]) {
+    super(...args);
+
+    1 + 2;
+  }
+
+  // eslint-disable-next-line require-jsdoc
+  public async testRun(): Promise<string> {
+    const file = await this.downloadAndDecryptFile(
+      'https://files-api-a.wbx2.com/v1/spaces/32ee69e8-f614-4b1b-a9f9-e472ab338e15/contents/ff44c732-91da-4d20-8979-aee71a0194e4/versions/2b59d407e4a64e6c8d71ec5eb4886832/bytes?keyUri=kms%3A%2F%2Fcisco.com%2Fkeys%2F3a0be121-c8e6-4ddb-b8a5-912c05b22c40&JWE=eyJlbmMiOiJBMjU2R0NNIiwiYWxnIjoiZGlyIn0..JnhlKeq14qXPLsKi.qDnQZFIyB7UCh1VxF53h42pWfQ1jnP6ZcwixnM9zFW9z4xMaKr3a-pLH_OEZNUOHM7A-qsoJHVvnACWNDNXfwPnKAMDERaS-9GsA3_mGqiT-CqgAWXrnkMSe1I3NHd3Q80RZ2zQ6TwxG9XrKUMgDvOWWp3oAc2UPJUDPJY04mAmK4LLXGrSJLENSuyf6IoofrBCtQMJGIw_vkwoA_9_uq0gK9JqSh-tjO-N7RBi51DyGmJko0Yr7Hb_QQgm2tv0eBVvQal0pIRJEPWmFTxu61XBvd2giOuRxDshG_Kmdf8MbngIdiLUcOfhosaNSFcVRzZYs_ZH4W0973QJCFw4-QFMOlL2rRztBQebgTTuxFPC6JouXth150y88BBBq-gbDzM__BAYITIsaR4v0YkPoAckF-9qvM4S2fqJOCKslCzf7u5yxEs9p8eVP1roVofM.dp2VT2zAPY1OUD-r-pOpGQ'
+    );
+    window.open(URL.createObjectURL(file));
+
+    return 'success';
+  }
+
+  /**
+   * Downloads and decrypts a file from the given URI.
+   *
+   * @param {string} fileUri - The URI of the file to be decrypted.
+   * @returns {Promise<File>} A promise that resolves to the decrypted file.
+   * @throws {Error} If the file download or decryption fails.
+   *
+   * @example
+   * ```typescript
+   * const decryptedFile = await downloadFile('https://example.com/encrypted-file');
+   * console.log(decryptedFile);
+   * ```
+   */
+  public async downloadAndDecryptFile(fileUri: string): Promise<File> {
```

Comment:
i have not seen the ticket but shouldnt we expose the encrypt and decrypt text logic also . are we planning to do that later in future sprint ?